src/linker/Linker.Steps/BodySubstituterStep.cs

```diff
@@ -0,0 +1,258 @@
+﻿using System;
+using System.IO;
+using System.Xml.XPath;
+using Mono.Cecil;
+using System.Globalization;
+
+namespace Mono.Linker.Steps
+{
+	public class BodySubstituterStep : BaseStep
+	{
+		protected override void Process ()
+		{
+			var files = Context.Substitutions;
```

Comment:
Why did you decide to go with a separate list of files instead of extending the ResolveFromXmlStep to be able to parse the `body` and `value` xml attributes?

Reply:
I can see pros and cons to both approaches.  Just curious what your thinking was.

Reply:
There is actually not that much in common and adding if everywhere would not make it readable